khasan.md

```diff
@@ -0,0 +1,5 @@
+#Toronto, ON
+
+- Belfast Love
```

Comment:
@khandkerhasan just throw a recommend craft 🍺 and I think this will be good to ⚓️